misc/test-extra/src/test/java/org/openscience/cdk/graph/invariant/HuLuIndexToolTest.java

```diff
@@ -0,0 +1,92 @@
+/*
+ * =====================================
+ *  Copyright (c) 2022 NextMove Software
+ * =====================================
```

Comment:
Copyright NextMove is fine. Please use the common format with the license, tho.

Reply:
Yes sorry just the default in the IDE :-)